chat-widget/src/common/Constants.ts

```diff
@@ -185,6 +185,12 @@ export class WebChatMiddlewareConstants {
     public static readonly adaptiveCard = "AdaptiveCard";
 }
 
+export class AMSConstants {
+    public static readonly supportedImagesMimeTypes = ["image/jpeg", "image/png", "image/gif", "image/heic", "image/webp"]; //using same MIME types as AMSClient
+    public static readonly maxSupportedImageSize = 20; //AMS max size for images is 20MB
+    public static readonly maxSupportedFileSize = 300; //AMS max size limit is 300MB but Dynamics limit is 128MB this gets overwritten on AttachmentUploadValidatorMiddleware.ts line 78
```

Comment:
Can we not specify the line number? The code base will evolve and the line number won't eventually be true anymore.

Reply:
`// AMS max file limit outside of supported Images MIME Types.`